test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -191,78 +200,100 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
                             tagHelper.FallbackTestClass = "hidden";
                             tagHelper.FallbackTestProperty = "visibility";
                             tagHelper.FallbackTestValue = "hidden";
-                        }
-                    },
-                    // File Version
-                    {
-                        new TagHelperAttributeList
-                        {
-                            ["asp-append-version"] = "true"
-                        },
-                        tagHelper =>
-                        {
                             tagHelper.AppendVersion = true;
                         }
```

Comment:
Ah. Actually this shouldn't be there. It makes no difference because there is no corresponding mock for `BuildUrlList`. That case is tested here `RendersLinkTags_GlobbedHref_AddsFileVersion`. Will remove this line.